pro/src/screens/OfferEducationalStock/OfferEducationalStock.tsx

```diff
@@ -0,0 +1,73 @@
+import { Formik } from 'formik'
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+import Banner from 'components/layout/Banner/Banner'
+import { SubmitButton, Title } from 'ui-kit'
+
+import FormStock from './FormStock'
+import styles from './OfferEducationalStock.module.scss'
+
+const OfferEducationalStock = (): JSX.Element => {
+  // TODO (Gautier, 2021-11-22) : À modifier au branchement du backend
+  const offerId = '1234'
+  const isOfferDraft = true
+  const editionOfferLink = `/offres/${offerId}/edition`
+  const offerIsDisbaled = false
+  const hasNoStock = false
+
+  return (
+    <>
+      <Title as="h2" level={4}>
+        Date et prix
+      </Title>
+      <Banner type="notification-info">
+        La date et le prix que vous indiquez sont à titre indicatif. Vous pouvez
+        par la suite définir toutes les dates réelles avec les établissements
+        scolaires.
+        <br />
+        Suite à votre événement, vous recevrez un mail pour confirmer les
+        éléments de votre offre et mettre à jour le nombre de participants
+        réels.
+      </Banner>
+      <p>
+        Indiquez le prix total du nombre de places maximum pour un établissement
+        scolaire.
+        <br />
+        (ex : mon offre s'élève à 200 euros pour 30 places dédiées à un
+        établissement scolaire.)
+      </p>
+      <div className={styles['educational-stock-form']}>
+        <Formik
+          initialValues={{}}
+          onSubmit={() => {
+            return
+          }}
+        >
+          {() => {
+            return <FormStock />
+          }}
```

Comment:
```suggestion
          {() =>  <FormStock />}
```

c'est du détail mais c'est plus court :p